Automattic/wp-calypso
# Playwright: write new media upload test cases

#### Changes proposed in this Pull Request

This PR adds new test cases not present in the older Selenium suite.

- new tests to check file upload functionality from Media gallery of various types (image, audio currently implemented).
- helper functions to generate test files.

#### Testing instructions

TeamCity
- ensure tests for Playwright Desktop E2E tests pass for this branch.

Local
- pull changes
- transpile code
- run `mocha --config .mocharc_playwright.yml` to verify the entire suite passes.

Depends on #53724
